tests/test_VideoFileClip.py

```diff
@@ -45,5 +46,28 @@ def test_ffmpeg_resizing():
         video.close()
 
 
+def test_video_mask():
+    """Test clip mask attributes operate."""
+    test_file = "resource/sintel_with_15_chapters.mp4"
+
+    return
```

Comment:
this makes the test does nothing, right?

Reply:
I didn't do this intentionally, but during the test, there will be an error that the resource/sintel_with_15_chapters.mp4 file cannot be found, which is a bit confusing. I don't know why this file cannot be found. Do you know what caused this?

Reply:
I seem to know the solution to the problem that occurred during the test! I have rewritten the test script.